### PR TITLE
Surya volunteer link

### PR DESCRIPTION
**Pull request by suryasdas**

### CODE REVIEW

File: voluncheer/map/urls.py [hunk 1]
```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     path("", views.map, name="map"),
+    # path("",views.search_address, name ="search_address")
```
Comment (EdenWuyifan) [inline voluncheer/map/urls.py:7]: This should be removed

File: voluncheer/map/views.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from django.shortcuts import render
 
+# from django.conf import settings
```
Comment (EdenWuyifan) [inline voluncheer/map/views.py:3]: Same here

File: voluncheer/opportunityboard/tests_views.py [hunk 3]
```diff
@@ -140,3 +146,47 @@ def test_deregister_volunteer(self):
         deregister_volunteer(test_request, self.opp.pk)
         self.opp.refresh_from_db()
         self.assertEqual(self.opp.staffing, 9)
+
+
+class OrganizationViewTestCase(TestCase):
+    def setUp(self):
+        self.client = Client()
```
Comment (EdenWuyifan) [inline voluncheer/opportunityboard/tests_views.py:153]: There's an opportunityboard.unittest_setup class you could refer to, no need to add everything here

File: voluncheer/opportunityboard/tests_views.py [hunk 4]
```diff
@@ -140,3 +146,47 @@ def test_deregister_volunteer(self):
...
+            end=dt.datetime.now().time(),
+            address_1="Test address 1",
+            address_2="Test address 2",
+            longitude="55.5555",
+            latitude="44.4444",
+            staffing=5,
+            is_published=True,
+        )
+
+    # self.opportunity2 = Opportunity.objects.create(
```
Comment (EdenWuyifan) [inline voluncheer/opportunityboard/tests_views.py:176]: Need to be removed

**Review by EdenWuyifan — CHANGES_REQUESTED** [review]

File: voluncheer/opportunityboard/tests_views.py (hunk 4 above)
Comment (suryasdas) [reply voluncheer/opportunityboard/tests_views.py:176]: do I have to remove all those lines?

File: voluncheer/opportunityboard/tests_views.py (hunk 3 above)
Comment (suryasdas) [reply voluncheer/opportunityboard/tests_views.py:153]: the test case I have written is different from the one in unites_setup file

File: .vscode/settings.json [hunk 5]
```diff
@@ -3,7 +3,7 @@
   "python.linting.flake8Enabled": true,
   "python.linting.enabled": true,
   "[python]": {
-    "editor.defaultFormatter": "ms-python.black-formatter",
+    "editor.defaultFormatter": "ms-python.python",
```
[the remark this answers is not in the document]
Comment (suryasdas) [reply .vscode/settings.json]: fixed it

File: voluncheer/templates/profiles/profile.html [hunk 6]
```diff
@@ -25,7 +25,11 @@
                 <hr align="center">
             </div>
             <div class="url">
+
```
[the remark this answers is not in the document]
Comment (suryasdas) [reply voluncheer/templates/profiles/profile.html:28]: fixed it

File: voluncheer/map/views.py (hunk 2 above)
Comment (suryasdas) [reply voluncheer/map/views.py]: fixed it

File: voluncheer/map/urls.py (hunk 1 above)
Comment (suryasdas) [reply voluncheer/map/urls.py]: done

**Review by EdenWuyifan — APPROVED** [review]
Approved, please rebase and squash

**Pull request merged**